### ISSUE TITLE
Ability to export/save and then reload results into the UI

### ISSUE THREAD

**Issue by p013570** [gchq/gaffer-tools#788]
The problem is really for users who spend a while querying and exploring a gaffer graph in the UI then they close their browser and have to recreate it all over again.

Currently there is a button to download the data to CSV from the data view, but there is no follow on button to reload it from a CSV.

The outcome of this ticket is the ability to close your browser, reopen it and reload your data quickly. 

The reload could be done by the user from a file they download, but this isn't required, the data could be stored somewhere else. Or it could just be the queries (and seeds) are stored somewhere and then rerun to reload the graph.

If this is done after #796 then the highlighted and renamed values should also be saved.

**Comment by d47853** [comment]
Merged into develop
